.github/workflows/build-release.yaml

```
@@ -42,4 +42,16 @@ jobs:
             *.xpi
             *.zip
             *.msi
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

```suggestion
      - uses: actions/checkout@v4
```

If there is no specific reason, it might better to use latest v4.